Remove bordercolordifference in darkthemeswitches

Feichtmeier opened the pull request:
When the border color of the top border is different than the rest of the border, there are dark and washed out edges in the border top where a darker green shines through the lighter green. Probably because gtk lets the background color shine through the green. which is no problem in the light theme but in the headerbar and the darktheme. We changed this already for the headerbar some months ago, this commit does the same for the dark theme in general.

Before:
![image](https://user-images.githubusercontent.com/15329494/57012869-d6c0ab80-6c08-11e9-80d3-3d87a93c2dcd.png)
![image](https://user-images.githubusercontent.com/15329494/57012918-02439600-6c09-11e9-8373-56de5e2b95c9.png)

![image](https://user-images.githubusercontent.com/15329494/57012893-edff9900-6c08-11e9-826d-9fa283f6db4a.png)


After:
![image](https://user-images.githubusercontent.com/15329494/57012842-b98bdd00-6c08-11e9-957a-916642f35420.png)
![image](https://user-images.githubusercontent.com/15329494/57012929-138ca280-6c09-11e9-8f55-7140bd484886.png)

![image](https://user-images.githubusercontent.com/15329494/57012830-ad078480-6c08-11e9-9896-f4b523d78e2f.png)

clobrano commented:
I agree on removing the top shadow, but why there is any difference between light and dark variant here? On headerbar they should be the same

Feichtmeier commented:
Oh you are right! Sorry, I look up my old commit and do the same for the dark theme!

Feichtmeier commented:
I also removed the border-top color difference, though in a stupid way by setting it to transparent:

https://github.com/ubuntu/yaru/pull/1139/commits/f04df32d6cf21c588ef98b33e1fb778a347adef6#diff-1b30fa3f60328fce0c66d39158906ac1R1652

> but why there is any difference between light and dark variant here

Because the dark background of the dark theme shines through the switch.
Actually I don't see many different ways here if we want a sharp switch in the dark theme. 
But my previous PR was a little bit strange, since the problem is the top border and not the box-shadow.

clobrano commented:
What I meant is that I noticed that the swtich is different in dark and light, not just for the border or shadows

![Screenshot from 2019-05-02 17-21-38](https://user-images.githubusercontent.com/2883614/57086782-1073de00-6cff-11e9-8860-e74017993b8b.png)
![Screenshot from 2019-05-02 17-21-59](https://user-images.githubusercontent.com/2883614/57086787-136ece80-6cff-11e9-8af5-c8beb840dc77.png)

specifically, the nob is smaller and less flat and I wondered why there is any difference since the css should be the same. I'll have a deeper look though

Feichtmeier commented:
Because I've set the border-top to transparent, which makes no sense. I push a fix

Feichtmeier commented:
@clobrano and/or ( @madsrh and/or @ubuntujaggers ) please test it again

The difference of the knob came from border-box / padding-box styling
If one/all/at least two of you prefer the bigger knob it would be border-box, padding-box if you prefer the smaller.

Moved the code out of common into drawing and
removed the border-color "bug" for the dark theme shown above

clobrano commented:
maybe a +1 for the bigger nob, but not important, now it looks good :+1:

Feichtmeier commented:
Bigger knob is closer to the shell switch so this might be an argument for the bigger knob
